### PR TITLE
Optimized imports in the chapter-remoting

### PR DESCRIPTION
**Pull request by WillemJiang**

### CODE REVIEW

File: chapter-remoting/src/main/scala/com/goticks/BoxOffice.scala [hunk 1]
```diff
@@ -1,11 +1,10 @@
 package com.goticks
 
-import scala.concurrent.duration._
-import scala.concurrent.Future
-
 import akka.actor._
 import akka.util.Timeout
 
+import scala.concurrent.Future
```
Comment (RayRoestenburg) [inline chapter-remoting/src/main/scala/com/goticks/BoxOffice.scala]: Please put scala imports first (before the akka.actor._ import)

File: chapter-remoting/src/main/scala/com/goticks/BoxOffice.scala [hunk 2]
```diff
@@ -56,8 +55,7 @@ class BoxOffice(implicit timeout: Timeout) extends Actor {
       context.child(event).fold(notFound())(getEvent)
 
     case GetEvents =>
-      import akka.pattern.ask
-      import akka.pattern.pipe
+      import akka.pattern.{ask, pipe}
```
Comment (RayRoestenburg) [inline chapter-remoting/src/main/scala/com/goticks/BoxOffice.scala]: Please add spaces between `{` and `ask` and `pipe` and `}`.

File: chapter-remoting/src/main/scala/com/goticks/BoxOffice.scala (hunk 1 above)
Comment (WillemJiang) [inline chapter-remoting/src/main/scala/com/goticks/BoxOffice.scala]: sure,  I will commit a quick fix for it.

File: chapter-remoting/src/main/scala/com/goticks/BoxOffice.scala (hunk 2 above)
Comment (WillemJiang) [inline chapter-remoting/src/main/scala/com/goticks/BoxOffice.scala]: OK.

**Comment by WillemJiang** [comment]
Hi Ray，
I just committed a quick fix as you suggested, please take a look at it.

Thanks,

Willem

**Comment by RayRoestenburg** [comment]
Thanks again @WillemJiang !

**Pull request merged**